libs/island-ui/core/src/lib/Box/useBoxStyles.treat.ts

```diff
@@ -588,6 +588,10 @@ export const opacity = styleMap({
   1: { opacity: 1 },
 })
 
+export const zIndex = styleMap({
+  1: { zIndex: 1 },
```

Comment:
Sounds very restrictive to only have `1` here. I can think of having 3 boxes with different z-index to pass on top of each other. It's probably good to not allow everything so we can keep big z-indexes for overlay/modal and so on, but maybe allowing a few more values could be nice? `10 - 90`?